resources/views/bread/read.blade.php

```diff
@@ -56,15 +56,14 @@
                                 <img class="img-responsive"
                                      src="{{ filter_var($dataTypeContent->{$row->field}, FILTER_VALIDATE_URL) ? $dataTypeContent->{$row->field} : Voyager::image($dataTypeContent->{$row->field}) }}">
                             @elseif($row->type == 'multiple_images')
-                                @if(json_decode($dataTypeContent->{$row->field}))
-                                    @foreach(json_decode($dataTypeContent->{$row->field}) as $file)
-                                        <img class="img-responsive"
-                                             src="{{ filter_var($file, FILTER_VALIDATE_URL) ? $file : Voyager::image($file) }}">
-                                    @endforeach
-                                @else
+                                @php $image_files = json_decode($dataTypeContent->{$row->field}) ?? []; @endphp
```

Comment:
Not sure if @emptynick agrees but I would remove this temporary variable and just have:
```php
@forelse (json_decode($dataTypeContent->{$row->field}) ?? [] as $file)
```
in the next line

Reply:
Doesn't really matter but I agree

Reply:
Wow! The elegance of this code is escalating fast, haha. Just updated the code.